### DISCUSSION TITLE
【V5】encode api文档链接报错404，如何隐藏encode 文本占位？

### DISCUSSION THREAD

**Discussion by PineSongCN** [antvis/G2#5240] [Q&A] [answered]
感谢您向我们反馈问题。

- **G2 Version**: 5.0.13
- **Platform**: Mac, Chrome
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

1. [encode api](https://g2.antv.antgroup.com/api/encode/overview) 文档链接报错404：
https://g2.antv.antgroup.com/api/encode/overview
![image](https://github.com/antvis/G2/assets/39976617/a22b8546-03cc-44a4-861b-ff9013ef38d8)

2. 请问如何隐藏encode的文本并去除占位空间？
[demo链接](https://g2.antv.antgroup.com/zh/examples/general/interval/#bar-stacked)
![image](https://github.com/antvis/G2/assets/39976617/294f22c5-3928-4737-a444-33390796eee5)

**Comment by pearmini** [comment]
```js
interval
  .axis('x', { title: false })
  .axis('y', { title: false })
  .legend('color', { title: false });
```

**Comment by PineSongCN** [comment]
> ```js
> interval
>   .axis('x', { title: false })
>   .axis('y', { title: false })
>   .legend('color', { title: false });
> ```

谢谢，这样是隐藏了title，但是占位还在，导致x、y轴和四边以及legend的间距明显的大一些

**Reply by pearmini** [reply, marked as answer]
`const chart = new Chart({padding: 'auto'})`

**Comment by PineSongCN** [comment]
> `const chart = new Chart({padding: 'auto'})`

感谢！另外请问文档访问的问题，是我访问的链接不对？还是V5的中文文档还在编写中？

**Reply by pearmini** [reply]
这个地方正在编写中，不是访问的问题。

**Comment by PineSongCN** [comment]
> 这个地方正在编写中，不是访问的问题。

谢谢，请问如何实现x轴title横向展示，如果显示不全，就省略部分列的title，左边是G2，右边是ECharts。
另外动态修改宽高的情况下，legend的位置计算会出现错误
![image](https://github.com/antvis/G2/assets/39976617/cbe7eab4-b495-44ab-a45a-80b0809d37f7)
